pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -207,16 +261,16 @@ func validateCustomPriorities(priorities map[string]config.PriorityPolicy, prior
 
 // validateExtendedResourceName checks whether the specified name is a valid
 // extended resource name.
-func validateExtendedResourceName(name v1.ResourceName) []error {
+func validateExtendedResourceName(path *field.Path, name v1.ResourceName) []error {
 	var validationErrors []error
 	for _, msg := range validation.IsQualifiedName(string(name)) {
-		validationErrors = append(validationErrors, errors.New(msg))
+		validationErrors = append(validationErrors, field.Invalid(path, name, msg))
 	}
 	if len(validationErrors) != 0 {
 		return validationErrors
 	}
 	if !v1helper.IsExtendedResourceName(name) {
-		validationErrors = append(validationErrors, fmt.Errorf("%s is an invalid extended resource name", name))
+		validationErrors = append(validationErrors, field.Invalid(path, string(name), fmt.Sprintf("%s is an invalid extended resource name", name)))
```

Comment:
`%s` is redundant. Just `is an invalid extended resource name` is enough.

Reply:
Done

Reply:
forgot to update the testcase as well...